swarm.go

```diff
@@ -261,21 +264,32 @@ func NewSwarm(config *api.Config, mockStore *mock.NodeStore) (self *Swarm, err e
 	self.bzzEth = bzzeth.New(self.netStore, to)
 
 	// Pss = postal service over swarm (devp2p over bzz)
-	self.ps, err = oldpss.New(to, config.OldPss)
+	self.oldpss, err = oldpss.New(to, config.OldPss)
 	if err != nil {
 		return nil, err
 	}
 	if oldpss.IsActiveHandshake {
-		oldpss.SetHandshakeController(self.ps, oldpss.NewHandshakeParams())
+		oldpss.SetHandshakeController(self.oldpss, oldpss.NewHandshakeParams())
 	}
 
 	if config.PushSyncEnabled {
 		// expire time for push-sync messages should be lower than regular chat-like messages to avoid network flooding
-		pubsub := oldpss.NewPubSub(self.ps, 20*time.Second)
+		pubsub := oldpss.NewPubSub(self.oldpss, 20*time.Second)
 		self.pushSync = pushsync.NewPusher(localStore, pubsub, self.tags)
 		self.storer = pushsync.NewStorer(self.netStore, pubsub)
 	}
 
+	self.pss = pss.NewPss(localStore)
+
+	if self.config.GlobalPinner {
```

Comment:
this is going ahead a bit, but why not... :)

Reply:
yeah, sorry, i just didn't want the callback and handler funcs set and registered without a condition.

we can continue iterating on this on the next phase